modules/virt-temporary-token-VNC.adoc

```diff
@@ -27,14 +27,15 @@ $ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} --type json
 # ...
 ----
 
-. Generate a token by running the following command:
+. Generate a token:
 +
 [source,terminal]
 ----
 $ curl --header "Authorization: Bearer ${TOKEN}" \
-     "https://api.<cluster_fqdn>/apis/token.kubevirt.io/v1alpha1/namespaces/<namespace>/virtualmachines/<vm_name>/vnc?duration=<duration>" <1>
+     "https://api.<cluster_fqdn>/apis/token.kubevirt.io/v1alpha1/namespaces/<namespace>/virtualmachines/<vm_name>/vnc?duration=<duration>"
```

Comment:
This structure apparently works if you "comment out the callout(s) in the codeblock". See the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).

```suggestion
     "https://api.<cluster_fqdn>/apis/token.kubevirt.io/v1alpha1/namespaces/<namespace>/virtualmachines/<vm_name>/vnc?duration=<duration>" // <1>
```